dev/parm/config/gfs/config.resources.GAEAC6

```diff
@@ -86,6 +86,38 @@ case ${step} in
         ;;
     esac
     ;;
+
+  "wavepostbndpnt")
+    walltime="01:30:00"
+    ntasks=1
+    threads_per_task=1
+    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export is_exclusive=True
+    NTASKS=${ntasks}
+    ;;
+
+  "wavepostbndpntbll")
+    walltime="00:25:00"
+    ntasks=2
+    threads_per_task=1
+    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export is_exclusive=True
+    NTASKS=${ntasks}
+    ;;
+
+  "wavepostpnt")
+    walltime="00:45:00"
+    ntasks=3
+    threads_per_task=1
+    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export is_exclusive=True
+    if [[ ${tasks_per_node} -gt 40 ]]; then
+        tasks_per_node=40
+        export is_exclusive=False
+    fi
+    NTASKS=${ntasks}
+    ;;
```

Comment:
I'm OK with these for now, but as you mention in the PR description, these will be tuned after spack-stack is upgraded.  Also, I wonder if these should be resolution-dependent and expanded to all machines (at least the walltime).  Can you add a `TODO:` to investigate/tweak the resources?

Reply:
I made an issue here: https://github.com/NOAA-EMC/global-workflow/issues/3883 to address this in a more holistic fashion and some things that had been in myhead as to-do's anyways.  Is that sufficient or do you want something in the code as well?